CMakeLists.txt

```diff
@@ -3,9 +3,11 @@ project(Elastix)
 
 set(Elastix_LIBRARIES elastix transformix)
 set(ELASTIX_BUILD_EXECUTABLE OFF CACHE BOOL "Generate executable or library")
-# Avoid LGPL code
-set(USE_KNNGraphAlphaMutualInformationMetric OFF)
-set(USE_KNNGraphAlphaMutualInformationMetric OFF CACHE BOOL "Use KNN metric.  Requires ANN library.")
+# Avoid LGPL code and ANN shared library
+if(NOT DEFINED USE_KNNGraphAlphaMutualInformationMetric)
```

Comment:
If `USE_KNNGraphAlphaMutualInformationMetric` exposed in UI?

Reply:
Yes, it is one of the advanced CMake options.